docs/api.apib

```diff
@@ -21,7 +21,7 @@ send events to the data broker or execute a given javascript function.
     + Body
 
             {
-                "time": "0/5 * * * *",
+                "time": "*/5 * * * *",
```

Comment:
If I'm not wrong, the cron's expressions accepted by the library we use starts on second not minute. So, the right expression should be `"0 */5 0 0 0 0 "`

Reply:
Leaving it off, the default will be 0, but for the sake of understanding the possibility of using seconds, I will add the seconds field.